src/TurbulenceClosures/turbulence_closure_implementations/isopycnal_skew_symmetric_diffusivity.jl

```diff
@@ -186,16 +185,15 @@ end
 # defined at cfc
 @inline function diffusive_flux_y(i, j, k, grid,
                                   closure::Union{ISSD, ISSDVector}, diffusivity_fields, ::Val{tracer_index},
-                                  velocities, tracers, clock, buoyancy) where tracer_index
+                                  c, clock, fields, buoyancy) where tracer_index
```

Comment:
Same comment here with `c`. Can we write out what `c` is in the calls to `diffusive_flux` to make it easier to understand? Is it tracers?